packages/app-runtime/src/modules/appEvents/RelationshipTemplateProcessedModule.ts

```diff
@@ -21,6 +21,10 @@ export class RelationshipTemplateProcessedModule extends AppRuntimeModule<Relati
 
         const data = event.data;
         switch (data.result) {
+            case RelationshipTemplateProcessedResult.RequestAutomaticallyDecided: {
+                break;
+            }
```

Comment:
I would put this to the last place (it is the most irrelevant case for the reader)